VERSION "1.0"

NS_ :
    NS_DESC_
    CM_
    BA_DEF_
    BA_
    VAL_
    SIG_VALTYPE_
    BO_TX_BU_
    BU_BO_REL_
    EV_DATA_
    ENVVAR_DATA_
    SG_MUL_VAL_
    SIG_GROUP_
    SIG_TYPE_REF_
    SIG_TYPE_
    SGTYPE_
    BO_REL_
    BU_SG_REL_
    BU_EV_REL_
    BU_BO_REL_
    BU_BO_REL_
    BU_BO_REL_
    BA_DEF_DEF_
    SG_MUL_VAL_

BS_:

BU_: ECU

BO_ 1345 HuynhDai_Elantra_Signal: 8 ECU
    SG_ LeftTurnSwitch : 19|1@0+ (1,0) [0|1] "" ECU
    SG_ RightTurnSwitch : 62|1@0+ (1,0) [0|1] "" ECU
    SG_ Flasher : 51|1@0+ (1,0) [0|1] "" ECU
    SG_ Hazard : 40|1@0+ (1,0) [0|1] "" ECU

VAL_ 1345 LeftTurnSwitch 0 "OFF" 1 "ON";
VAL_ 1345 RightTurnSwitch 0 "OFF" 1 "ON";
VAL_ 1345 Flasher 0 "OFF" 1 "ON";
VAL_ 1345 Hazard 0 "OFF" 1 "ON";
